main.go

```diff
@@ -81,6 +82,7 @@ func main() {
 	var serviceName string
 	var serviceNamespace string
 	targetLabels := []vulnerabilityreport.VulnerabilityLabel{}
+	reportLabels := []ciskubebenchreport.ReportLabel{}
```

Comment:
Same here, these should be made clearer (the vulnerability targetLabels should be renamed too but one of us can do that in a followup)
```suggestion
	cisDetailReportLabels := []ciskubebenchreport.ReportLabel{}
```